src/maga/Game.java

```diff
@@ -279,15 +285,30 @@ private boolean youWin(){
         long endTime = System.currentTimeMillis() / 1000L;
         long elapsedTime = endTime - startTime;
         long finalScore = points - (elapsedTime * steps);
+        if (bonusTime == 0) {
+            Console.print(
+            "",
+            "Congratulations, you won the game!",
+            "",
+            "---------------------------------------",
+            "You made it in " + steps + " steps, in " + elapsedTime + " seconds!",
+            "You scored: " + finalScore,
+            "---------------------------------------"
+        );
+        } else {
+            finalScore = points - ((elapsedTime - bonusTime) * steps);
         Console.print(
             "",
             "Congratulations, you won the game!",
             "",
             "---------------------------------------",
             "You made it in " + steps + " steps, in " + elapsedTime + " seconds!",
+            "You received a time bonus " + bonusTime + " seconds, for calling Trump!",
```

Comment:
```java
bonusTime > 0 ? "You made it in " + steps + " steps, in " + elapsedTime + " seconds!" : null
```

Reply:
fixed in: 2bac00b3d2c17390fdd52f1b7a3caf376f5939a2